app/Console/Commands/Cloud/CloudFixSubscription.php

```
@@ -0,0 +1,879 @@
+        $message .= "Subscription ID: {$subscription->stripe_subscription_id}\n";
+
+        send_internal_notification($message);
+
+        // Call the team's subscription ended method which properly cleans up
+        $team->subscriptionEnded();
+    }
+
+    /**
+     * Search for subscriptions by customer ID
+     */
+    private function searchSubscriptionsByCustomer(\Stripe\StripeClient $stripe, $customerId, $requireActive = false)
```

PHPMD (2.15.0)

The method searchSubscriptionsByCustomer has a boolean flag argument $requireActive, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)